definition/data/sheets/CaseField.json

```diff
@@ -92,5 +92,6 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "directionOrder", "Label": "Direction order", "FieldType": "DirectionOrder", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "channel", "Label": "Channel", "FieldType": "FixedList", "FieldTypeParameter": "ChannelType", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "intentionToProceedDeadline", "Label": "Intention To Proceed Deadline", "FieldType": "Date", "FieldTypeParameter": "ChannelType", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "state", "Label": "state", "FieldType": "Text", "SecurityClassification": "Public"}
+  {"CaseTypeID": "MoneyClaimCase", "ID": "state", "Label": "state", "FieldType": "Text", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "RPAEventType", "Label": "Resend RPA", "FieldType": "FixedList", "FieldTypeParameter": "RpaType", "SecurityClassification": "Public"}
```

Comment:
This is not required in my opinion to store RPAType that is retriggered. If we need it, there are many times we an retrigger the RPA, so we have to log all the time we have sent RPA.

Reply:
if we are populating this field in about to start and showing in page, then this field should be here.